test/blackbox-tests/test-cases/dune-cache/trim.t/run.t

```diff
@@ -83,15 +85,20 @@ entries uniformly.
   $ dune_cmd stat size "$PWD/.xdg-cache/dune/db/meta/v5/71/71a631749bd743e4c107ba109224c12f"
   70
 
-Trimming the cache at this point should not remove anything because all file
-entries are still hard-linked from the build directory.
+Trimming the cache at this point should not remove any file entries because all
+of them are still hard-linked from the build directory. However, we should trim
+all metadata entries in [meta/v4] since they are broken: remember, we moved all
+[files/v4] to [files/v3].
 
+  $ find "$PWD/.xdg-cache/dune/db/meta/v4" -mindepth 2 -maxdepth 2 -type f -printf '+'
```

Comment:
Unfortunately, `-printf` doesn't work in a BSD find (and by extension Mac). Perhaps rewrite using `wc`, or `xargs`? Or even implement the thing in OCaml and put it in `dune_cmd`.

Reply:
Yeah, I started with `wc` but its output was non-reproducible too (there were some tabs). I'll figure something out.

Reply:
`-exec 'printf +'` should be more portable

Reply:
Oh, there is already `dune_cmd count-lines`. Cool!